doc/src/dhwheater-doc.md

```diff
@@ -284,18 +289,70 @@ Air source heat pump type, valid only if whHeatSrc=ASHPX. These choices are supp
 "BasicIntegrated","Typical integrated HPWH"
 "ResTank","Resistance heater (no compressor).  Superceded by whHeatSrc=RESITANCEX"
 "ResTankNoUA","Resistance heater (no compressor) with no tank losses.  Superseded by whHeatSrc=RESISTANCEX."
 "AOSmithHPTU80DR","80 gallon AOSmith HPTU with fixed backup setpoint (experimental for demand response testing)"
 "AOSmithSHPT50","50 gal AOSmith SHPT"
 "AOSmithSHPT66","66 gal AOSmith SHPT"
 "AOSmithSHPT80","80 gal AOSmith SHPT"
+"ColmacCxV5_SP","Colmac CxA-xx modular external HPWHs (single pass mode)"
+"ColmacCxA10_SP","Colmac CxA-xx modular external HPWHs (single pass mode)"
+"ColmacCxA15_SP","Colmac CxA-xx modular external HPWHs (single pass mode)"
+"ColmacCxA20_SP","Colmac CxA-xx modular external HPWHs (single pass mode)"
+"ColmacCxA25_SP","Colmac CxA-xx modular external HPWHs (single pass mode)"
+"ColmacCxA30_SP","Colmac CxA-xx modular external HPWHs (single pass mode)"
+"ColmacCxV5_MP","Colmac CxA-xx modular external HPWHs (multi-pass mode)"
+"ColmacCxA10_MP","Colmac CxA-xx modular external HPWHs (multi-pass mode)"
+"ColmacCxA15_MP","Colmac CxA-xx modular external HPWHs (multi-pass mode)"
+"ColmacCxA20_MP","Colmac CxA-xx modular external HPWHs (multi-pass mode)"
+"ColmacCxA25_MP","Colmac CxA-xx modular external HPWHs (multi-pass mode)"
+"ColmacCxA30_MP","Colmac CxA-xx modular external HPWHs (multi-pass mode)"
+"NyleC25A_SP","Nyle Cxx external HPWHs (SP = single pass mode)"
+"NyleC60A_SP","Nyle Cxx external HPWHs (SP = single pass mode)"
+"NyleC90A_SP","Nyle Cxx external HPWHs (SP = single pass mode)"
+"NyleC125A_SP","Nyle Cxx external HPWHs (SP = single pass mode)"
+"NyleC185A_SP","Nyle Cxx external HPWHs (SP = single pass mode)"
+"NyleC250A_SP","Nyle Cxx external HPWHs (SP = single pass mode)"
+"NyleC60A_CWP_SP","Nyle Cxx external SP HPWHs with cold weather package"
+"NyleC90A_CWP_SP","Nyle Cxx external SP HPWHs with cold weather package"
+"NyleC125A_CWP_SP","Nyle Cxx external SP HPWHs with cold weather package"
+"NyleC185A_CWP_SP","Nyle Cxx external SP HPWHs with cold weather package"
+"NyleC250A_CWP_SP","Nyle Cxx external SP HPWHs with cold weather package"
+"NyleC60A_MP","Nyle Cxx external HPWHs (MP = multi-pass mode)"
+"NyleC90A_MP","Nyle Cxx external HPWHs (MP = multi-pass mode)"
+"NyleC125A_MP","Nyle Cxx external HPWHs (MP = multi-pass mode)"
+"NyleC185A_MP","Nyle Cxx external HPWHs (MP = multi-pass mode)"
+"NyleC250A_MP","Nyle Cxx external HPWHs (MP = multi-pass mode)"
+"NyleC60A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC90A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC125A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC185A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"NyleC250A_CWP_MP","Nyle Cxx external MP HPWHs w/ cold weather package"
+"Rheem_HPHD60HNU_MP","???"
+"Rheem_HPHD60VNU_MP","???"
+"Rheem_HPHD135HNU_MP","???"
+"Rheem_HPHD135VNU_MP","???"
```

Comment:
@pmskintner @chipbarnaby Are these used? Can we get descriptions for them?

Reply:
To keep the same format as other descriptions, something like: "Rheem HPHDxx MP external MP HPWHs"

Reply:
Agree with Paul, this list has to be trued up with CNDTYPES.DEF.  The Rheem models are defined, but I would defer to Paul as to how to describe them.